content/en/docs/contribute/blog/mirroring.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Blog article mirroring
+slug: article-mirroring
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+There are two official Kubernetes blogs, and the CNCF has its own blog where you can cover Kubernetes too.
+For the main Kubernetes blog, we (the Kubernetes project) like to publish articles with different perspectives and special focuses, that have a link to Kubernetes.
+
+Some articles appear on both blogs: there is a primary version of the article, and
+a _mirror article_ on the other blog.
+
+This page describes the criteria for mirroring, the motivation for mirroring, and
+explains what you should do to ensure that an article publishes to both blogs.
+
+# {{% heading "prerequisites" %}}
+
+Make sure you are familiar with the introduction sections of
+[contributing to Kubernetes blogs](/docs/contribute/blog/), not just to learn about
+the two official blogs and the differences between them, but also to get an overview
+of the process.
+
+<!-- content -->
+
+## Why we mirror
+
+Mirroring is nearly always from the contributor blog to the main blog. The project does this
+for articles that are about the contributor community, or a part of it, but are also relevant
+to the wider set of readers for Kubernetes' main blog.
+
+As an author (or reviewer), consider the target audience and whether the blog post is appropriate for the [main blog](/docs/contribute/blog/#main-blog).
+For example: if the target audience are Kubernetes contributors only, then the
+[contributor blog](/docs/contribute/blog/#contributor-blog).
+may be more appropriate;
+if the blog post is about open source in general then it may be more suitable on another site outside the Kubernetes project.
+
+This consideration about target audience applies to original and mirrored articles equally.
+
+The Kubernetes project is willing to mirror any blog article that was published to https://kubernetes.dev/blog/
```

Comment:
Proper markdown should be as follows in case you want to represent a clickable link:

```suggestion
The Kubernetes project is willing to mirror any blog article that was published to <https://kubernetes.dev/blog/>
```